_source/_posts/2021-06-07-session-mgmt-node.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: blog_post
+title: "Everything You Ever Wanted to Know About Session Management in Node.js"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "A quick run-down of how to manage sessions in Node.js."
+tags: [session, node]
+tweets:
+- "Need to learn how to manage sessions in Node? We've got you covered!"
+- "Learn how to manage sessions in Node!"
+- "Managing sessions in Node can be confusing. Check this out!"
+image: blog/featured/okta-node-skew.jpg
+type: conversion
+---
+
+Session Management is a pretty scary term for a lot of developers.  Most of the mechanics of session management are abstracted away from developers, to the point where they don't properly learn about it until it's necessary.  Usually, this means a late night trying to figure out a vulnerability, a bug, or how to work with a new session management library.  Hopefully, I can remove some of the magic behind session management in NodeJs and give you a good fundamental place to start.  In this article, you will learn what sessions are, how to manage them in Node.js, and some details that should help you minimize bugs and vulnerabilities.
+
+## Overview of Sessions
```

Comment:
It seems the keywords are session and node. If that's what you're targeting, it seems like Node.js should be in the headings more.